### DISCUSSION TITLE
Please become Brave Creator so that Subtitle Edit users can make donations in $BAT

### DISCUSSION THREAD

**Discussion by zcraber** [SubtitleEdit/subtitleedit#5420] [General]
@niksedk Please consider becoming a [Brave Creator](https://publishers.basicattentiontoken.org/) by adding your website and GitHub page of Subtitle Edit so that users from the [Brave browser](https://brave.com/) can tip you with crypto token $BAT. It's free and super easy to set up.

**Comment by darnn** [comment]
Counterpoint: Please don't. Crypto is evil and scammy.
This article, while focusing more on NFTs, gives a pretty good rundown:
https://everestpipkin.medium.com/but-the-environmental-issues-with-cryptoart-1128ef72e6a3

**Reply by zcraber** [reply]
> Counterpoint: Please don't. Crypto is evil and scammy. This article, while focusing more on NFTs, gives a pretty good rundown: https://everestpipkin.medium.com/but-the-environmental-issues-with-cryptoart-1128ef72e6a3

Not all cryptos are bad for the environment. BAT relies on the ETH blockchain and it doesn't directly affect energy consumption. Moreover, ETH is moving to [PoS](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/) from PoW which will be safe for the environment.

**Comment by darnn** [comment]
To quote the title of the article:
HERE IS THE ARTICLE YOU CAN SEND TO PEOPLE WHEN THEY SAY “BUT THE ENVIRONMENTAL ISSUES WITH CRYPTOART WILL BE SOLVED SOON, RIGHT?”
Cryptocurrencies and NFTs are an absolute disaster for so many more reasons than the ecological.

**Reply by zcraber** [reply]
> To quote the title of the article: HERE IS THE ARTICLE YOU CAN SEND TO PEOPLE WHEN THEY SAY “BUT THE ENVIRONMENTAL ISSUES WITH CRYPTOART WILL BE SOLVED SOON, RIGHT?” Cryptocurrencies and NFTs are an absolute disaster for so many more reasons than the ecological.

Do you really think fiat, banking, gold etc have[ no impact](https://www.nasdaq.com/articles/a-comparison-of-bitcoins-environmental-impact-with-that-of-gold-and-banking-2021-05-04)? It is much worse than crypto. 
Honestly, idk why you're bringing up NFTs for discussion here. It has nothing to do with my suggestion which is fyi BAT.

**Comment by darnn** [comment]
I'm bringing them up because, as I said, most of that stuff applies to crypto in general, and it's the article I have on hand. Regardless, I'm not trying to convince _you_, I'm just trying to note the fact that I think this is a terrible idea.
